pandas/tests/test_categorical.py

```diff
@@ -3163,18 +3163,6 @@ def test_info(self):
         buf = compat.StringIO()
         df2.info(buf=buf)
 
-    def test_groupby_sort(self):
```

Comment:
This and test_groupby were moved to `tests/categorical/test_groupby.py`. `test_groupby` was updated for the API change.